docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -263,17 +234,15 @@ Q16. What does the message "Input shape is required to convert MXNet model. Plea
 
 .. _question-17:
 
-Q17. What does the message "Both --mean_file and mean_values are specified. Specify either mean file or mean values" mean?
 #####################################################################################################################################################
 
-**A:** The ``--mean_file`` and ``--mean_values`` options are two ways of specifying preprocessing for the input. However, they cannot be used together, as it would mean double subtraction and lead to ambiguity. Choose one of these options and pass it with the corresponding CLI option.
+This MO argument/option was deprecated and removed, question is not actual anymore.
```

Comment:
Are Q17-18 planned to be removed too? If not, let's maybe change the word actual to something like - This MO option/argument has been deprecated, the question is no longer relevant.

Reply:
Q17-18 should also be removed. this green lines are left by mistake, i will remove them as well

Reply:
removed these lines